ural/instagram.py

```diff
@@ -14,17 +14,19 @@
 INSTAGRAM_USERNAME_RE = re.compile(r"^[a-zA-Z0-9_\-\.]+$")
 INSTAGRAM_DOMAIN_RE = re.compile(r"instagram.com$", re.I)
 INSTAGRAM_URL_RE = re.compile(DOMAIN_TEMPLATE % r"(?:[^.]+\.)*instagram.com", re.I)
+INSTAGRAM_NOT_A_USER_LIST = ["accounts", "ads", "direct", "emails", "explore", "push", "reels", "session", "settings", "stories", "your_activity"]
 
 InstagramUser = namedtuple("InstagramUser", ["name"])
 InstagramPost = namedtuple("InstagramPost", ["id", "name"])
+InstagramReel = namedtuple("InstagramReel", ["id"])
 
 
 def is_instagram_post_shortcode(value):
-    return bool(re.search(INSTAGRAM_POST_SHORTCODE_RE, value))
+    return bool(re.search(INSTAGRAM_POST_SHORTCODE_RE, value)) and value != "videos"
```

Comment:
Conceptuellement ce `and` n'a pas sa place dans cette fonction. `videos` **est** un shortcode de post valide. Le fait de vérifier si c'est `videos` n'a de sens que quand on considère un path d'url, donc dans la fonction de parse, à mon sens. Tu ne trouves pas?